kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestApplication.java

```diff
@@ -93,13 +93,11 @@ protected void setupInjectedResources(
       ProducerPool producerPool,
       KafkaConsumerManager kafkaConsumerManager
   ) {
-    if (StringUtil.isBlank(appConfig.getString(KafkaRestConfig.BOOTSTRAP_SERVERS_CONFIG))
-        && StringUtil.isBlank(appConfig.getString(KafkaRestConfig.ZOOKEEPER_CONNECT_CONFIG))) {
-      throw new RuntimeException("Atleast one of " + KafkaRestConfig.BOOTSTRAP_SERVERS_CONFIG + " "
-                                 + "or "
-                                    + KafkaRestConfig.ZOOKEEPER_CONNECT_CONFIG
-                                    + " needs to be configured");
+    /* TODO: Update EmbeddedServerTestHarness to allow required BOOTSTRAP_SERVERS_CONFIG */
```

Comment:
Let's not have TODOs in the code please. If this needs to be done as a follow-up, we should file a JIRA and track it that way.

Reply:
Ack, will remove.